Dockerfile.ia-etl

```diff
@@ -0,0 +1,31 @@
+# Use an official Python runtime as a parent image
+FROM python:3.6-slim
+MAINTAINER enviroDGI@gmail.com
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    git gcc g++
+
+# Set the working directory to /app
+WORKDIR /app
+ENV WEB_MONITORING_DB_URL
+ENV WEB_MONITORING_DB_PASSWORD
+ENV WEB_MONITORING_DB_EMAIL
+ENV WEB_MONITORING_FROM
+ENV WEB_MONITORING_PARALLEL
```

Comment:
I think these should both have defaults (and those defaults can be the same values you had before) — I just want the vars so that they are customizable without creating a whole new image.

I’m also not sure there’s any need to namespace these as `WEB_MONITORING_*`. The other ones have that in the name because they are referring to an instance of web-monitoring-db, not because they are being namespaced. For example, we also have `VALIDATE_TARGET_CERTIFICATES`, which is not namespaced: https://github.com/edgi-govdata-archiving/web-monitoring-processing/blob/9a8430c6385d9d5b384417eaeb23d06d99332563/.env.example#L32-L35